Formula/h/hdf5-mpi.rb

```diff
@@ -112,5 +109,9 @@ def install
     EOS
     system "#{bin}/h5pfc", "test.f90"
     assert_equal version.to_s, shell_output("./a.out").chomp
+
+    # Make sure that it was built with SZIP/libaec
+    config = shell_output("#{bin}/h5cc -showconfig")
+    assert_match %r{I/O filters.*DECODE}, config
```

Comment:
Please also make sure it's build with MPI

Reply:
It's already using the parallel compiler wrappers (`h5pcc` instead of `h5cc` and `h5pfc` instead of `h5fc`). Is that enough? Or should I check via preprocessor for `HAVE_H5_PARALLEL`?